share/timing/GPTLget_memusage.c

```diff
@@ -108,7 +108,7 @@ int GPTLget_memusage (int *size, int *rss, int *share, int *text, int *datastack
 #elif (defined HAVE_SLASHPROC)
   FILE *fd;                       /* file descriptor for fopen */
   int pid;                        /* process id */
-  char file[19];                  /* full path to file in /proc */
+  char file[20];                  /* full path to file in /proc */
```

Comment:
Is an `int`'s size really set to 8 bytes in the build flags for this code? In C/C++, an int is almost always 4 bytes.

Reply:
> Although this may not have a performance impact, I agree with bumping to 24. Along with preceding fields, 24 aligns with 8 byte boundary.
> 
> ```
> sizeof(FILE *) : 8
> sizeof(int): 8
> ```

OK, I will bump it to 24 then, thanks.

Reply:
For int size 8, I did a cursory test with the GNU on Summit. It may vary based on compiler defaults. Anyway, it's ok to bump to 24.